plugins/notifications-backend/migrations/20240531_metadata.js

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+exports.up = async function up(knex) {
+  await knex.schema.createTable('notification_metadata', table => {
```

Comment:
Are these commonly amended or removed etc? Maybe this has been discussed elsewhere, but spontaneously I'd think it fine to slap this into just another json string.

Reply:
Are you talking about the whole table?

I started with json string, but then @Rugvip came up with this structure https://github.com/backstage/backstage/pull/24562#discussion_r1601491520